samples/fhir-to-fhir/api-for-fhir-to-api-for-fhir/docs/FAQ.md

```diff
@@ -0,0 +1,104 @@
+# FAQ's for FHIR-to-FHIR data movement
+
+1. How do we perform FHIR to FHIR (Gen1 to Gen1) data transfer?
+   + For implementation details and architecture overview please refer the [documentation](https://github.com/Azure-Samples/azure-health-data-services-samples/tree/main/samples/fhir-to-fhir/api-for-fhir-to-api-for-fhir#copying-data-from-one-azure-api-for-fhir-server-to-another-azure-api-for-fhir-server).
+
+2. How to deploy FHIR loader?
```

Comment:
It would be good to call out the restrictions here. This will only work if you have full access to create resources in your subscription and are not using private networking.

Reply:
changes done as per review comments.

Reply:
perfecttt